services/README.md

```diff
@@ -157,8 +157,8 @@ function uses_env {
   echo true/false
 }
 
-# add supported arch(es)
-function supported_arches {
+# add supported arm(s)
+function supported_arms {
```

Comment:
this works too

@dyeh123 but I wanna go in the other direction
I wanna rename all `supported_arms` everywhere into `supported_arches`